src/components/mdTabs/mdTab.vue

```diff
@@ -14,7 +14,9 @@
       mdLabel: [String, Number],
       mdIcon: String,
       mdActive: Boolean,
-      mdDisabled: Boolean
+      mdDisabled: Boolean,
+      mdTooltip: String,
+      mdTooltipDirection: String
```

Comment:
The direction default should be set here to not make the getTabData do the dirty check everytime.